### ISSUE TITLE
Crash if restart app from background by clicking icon on Android

### ISSUE THREAD

**Issue by minggo** [cocos2d/cocos2d-x#16520]

**Comment by minggo** [comment]
It is because by default some configuration changes will restart the activity, so we should prevent it by declare the events in AndroidMenifest.xml:

```
android:configChanges="orientation|keyboardHidden|screenSize"
```

`screenSize` is added since API 13, and API 13, so we should change the API level to 13 by default. Good news is that, if you are building with API 13, you can also run on API 10 because our java codes can work with API 10+.

**Comment by minggo** [comment]
fixed in #16525
